db/migrate/20241017192240_add_description_to_avatar_and_header.rb

```diff
@@ -0,0 +1,6 @@
+class AddDescriptionToAvatarAndHeader < ActiveRecord::Migration[7.1]
+  def change
+    add_column :accounts, :avatar_description, :string
+    add_column :accounts, :header_description, :string
```

Comment:
I think we may need defaults here to empty strings, but I can't recall how that will affect migration performance (probably a question for Claire)